Client.xcodeproj/project.pbxproj

```diff
@@ -13959,7 +13959,7 @@
 			repositoryURL = "https://github.com/mozilla/rust-components-swift";
 			requirement = {
 				kind = exactVersion;
-				version = 91.0.1;
+				version = 92.0.0;
```

Comment:
Should this be updated to `92.0.1` as it was done in https://github.com/mozilla-mobile/firefox-ios/pull/10334/files ?

Reply:
Good catch @lmarceau!! so I was confused why xcode didn't bump this on its own, and I realized there are two entries in the `.pbxprob` for `rust-components-swift`.

I'm not sure how it got here, but this one seems to have been outdated. It was on 91.0.1, when the [bottom one](https://github.com/mozilla-mobile/firefox-ios/pull/10297/files#diff-053ad7baef0cd20365abec6f2d6133fa7eed9440d908c9f92e1319c8d610d17fR13991) was correctly at 91.1.0. It seems to me that this one isn't actually used, so I deleted it and firefox built locally just fine

I'll update the PR... I'm still confused why xcode didn't clean this up on its own